build.gradle.kts

```diff
@@ -225,6 +224,35 @@ swaggerSources {
     }
 }
 
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    // limit format enforcement to just the files changed by this branch
+    ratchetFrom("origin/main")
+
+    format("misc") {
+        target("*.gradle", ".gitattributes", ".gitignore", ".containerignore", ".dockerignore")
+
+        trimTrailingWhitespace()
+        indentWithSpaces()
+        endWithNewline()
+    }
+    java {
+        targetExclude("**/src/generated/**", "**/build/generated/**")
```

Comment:
nice! that's still not all our generated files because they don't all end up in 'generated' packages right now but we can fix that. :-)

Reply:
Yep - that's one of the reasons I don't want to have this applied to all files. I don't have enough knowledge yet to add the rest of the clients :(

Reply:
ah clear. I can fix those clients so that they are all in 'generated' packages I think.